apps/src/submitHelper.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * This helper can be used by an app (e.g. GameLab) to handle the pressing of
+ * submit/unsubmit buttons if the level is submittable.
+ */
+
+import commonMsg from '@cdo/locale';
+import dom from './dom';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DialogButtons from './templates/DialogButtons';
+
+// Parameters provided by the calling app.
+var submitParams;
+
+/**
+ * Set up the handlers for the submit and unsubmit buttons.
+ * The DOM should have submitButton and unsubmitButton by now.
+ * Also store some parameters to be used in these handlers.
+ *
+ * @param {Object} params
+ * @param {Object} params.studioApp - The studioApp itself.
+ * @param {function} params.onPuzzleComplete - Function to call when submitting.
+ * @param {string} params.unsubmitUrl - URL to post to when unsubmitting.
+ */
+export function setupSubmitButtons(params) {
```

Comment:
I wonder if we can/should just import the studioApp singleton instead of passing it in